sympy/core/function.py

```diff
@@ -1805,7 +1805,25 @@ def mystr(expr, **settings):
     def _eval_is_commutative(self):
         return self.expr.is_commutative
 
+    def is_continuous(expression,symbol,points):
+        left_hand_limit = expression.subs(symbol,points-10**(-20))
+        right_hand_limit = expression.subs(symbol,points+10**(-20))
+        value=expression.subs(symbol.points)
+        if left_hand_limit == right_hand_limit and left_hand_limit == value:
```

Comment:
These are not going to be equal in most cases, which has no input on the discontinuity of the expression.